aiida/cmdline/commands/cmd_devel.py

```diff
@@ -20,26 +22,36 @@ def verdi_devel():
 
 
 @verdi_devel.command('check-load-time')
-def devel_check_load_time():
+@click.option('-v', '--verbose', is_flag=True, help='Increase verbosity')
+def devel_check_load_time(verbose):
     """Check for common indicators that slowdown `verdi`.
 
     Check for environment properties that negatively affect the responsiveness of the `verdi` command line interface.
     Known pathways that increase load time:
 
         * the database environment is loaded when it doesn't need to be
-        * the `aiida.orm` module is imported when it doesn't need to be
+        * Only expected `aiida.*` modules are imported
 
     If either of these conditions are true, the command will raise a critical error
     """
     from aiida.manage.manager import get_manager
 
+    aiida_modules = [key for key in sys.modules if key.startswith('aiida.')]
+    aiida_modules_str = '\n- '.join(sorted(aiida_modules))
+    if verbose:
+        echo.echo(f'aiida modules loaded:\n- {aiida_modules_str}')
+
     manager = get_manager()
 
     if manager.backend_loaded:
         echo.echo_critical('potential `verdi` speed problem: database backend is loaded.')
 
-    if 'aiida.orm' in sys.modules:
-        echo.echo_critical('potential `verdi` speed problem: `aiida.orm` module is imported.')
+    allowed = ('aiida.backends', 'aiida.cmdline', 'aiida.common', 'aiida.manage', 'aiida.plugins', 'aiida.restapi')
```

Comment:
I guess this is a constant, perhaps name it accordingly

Reply:
what name did you have in mind? I'm not fussed with whatever

Reply:
capital letters?

Reply:
I’d say capital is only for global variables. I can make it a global variable, but don’t think it’s that necessary